src/test/java/org/tarantool/ClientResultSetIT.java

```diff
@@ -0,0 +1,990 @@
+package org.tarantool;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.tarantool.conversion.NotConvertibleValueException;
+import org.tarantool.dsl.Requests;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+
+public class ClientResultSetIT {
+
+    private static TarantoolTestHelper testHelper;
+
+    private TarantoolClient client;
+
+    @BeforeAll
+    static void setupEnv() {
+        testHelper = new TarantoolTestHelper("client-resultset-it");
+        testHelper.createInstance();
+        testHelper.startInstance();
+    }
+
+    @BeforeEach
+    void setUp() {
+        client = TestUtils.makeTestClient(TestUtils.makeDefaultClientConfig(), 2000);
+    }
+
+    @AfterEach
+    void tearDown() {
+        client.close();
+    }
+
+    @AfterAll
+    static void tearDownEnv() {
+        testHelper.stopInstance();
+    }
+
+    @Test
+    void testGetSimpleRows() {
+        testHelper.executeLua(
```

Comment:
(not in this request) Consider using [test-containers](https://www.testcontainers.org/) for setting up the integration test environment.